packages/service-integration/requirements/_tools.in

```diff
@@ -5,4 +5,5 @@
 --requirement ../../../requirements/devenv.txt
 
 # to generate compose_spec_model.py
-datamodel-code-generator[http]
+# TODO: not essential, move to a container
+#datamodel-code-generator[http]
```

Comment:
remove that comment?

Reply:
No we put it here for reference. We need to take action upon it at a later date